examples/evp_1d_waves_on_a_string/waves_on_a_string.py

```diff
@@ -67,3 +67,11 @@
 plt.tight_layout()
 plt.savefig('eigenvalue_error.pdf')
 
+plt.figure(figsize=(6, 4))
+for n, idx in enumerate(np.argsort(solver.eigenvalues)[:5], start=1):
```

Comment:
`x` is not defined yet. Could add `x = dist.local_grid(xbasis)` before the for loop.